_posts/blog/2016-09-13-code-reviews.md

```diff
@@ -0,0 +1,91 @@
+
+By the way: using _please_ to soften a a harsh sentence does not help. You already clearly communicated a difference in status. You already switched your consultant-hat for something that is simply inappropriate in modern software development teams. The result will be a bad taste in the mouth at best.
+
+Even more, given as a command, even if correct, the advice won't be accepted wholeheartedly and the team morale is damaged.
+
+
+If the reviewer would have invested just a little more time and watched the way he communicated: the reviewee might have been very thankful and might have accepted all advise with a smile.
+
+
+The effort is well invested if one takes a break from time to time and makes sure the mindset is tuned, communication style is respectful and reflects the intention to help.
+
+
+## Don't Enforce Consensus
+
+Engineers often tend to forget that there is more than one valid opinion in a lot of cases. __Not everything is determined to false and true__.
+
+So if the reviewer has communicated his opinion appropriate and was not able to convince the reviewee, the code review is the wrong forum to let discussion escalate and enforce consensus. If the code is robust and fulfills the requirements, let it pass the review and try to move the topic out of the situation. You may have tech meetings or other forums in which you can discuss the topic (ideally in general and __not__ by specific example) with a broader audience.
+
+Letting the topic lie for a week can also help. Often times you realize that consensus __isn't important at all__. That not only there are more valid opinions than yours. But also that your team maybe can afford to __let both opinions stand__.
+
+## Don't Point At Rules - Give Explanations
+
+Rules and processes kill __creativity__. You want to give as much freedom to everyone as possible without hurting someone else (that's part of the German Constitution by the way ;)).
+
+Developers develop processes and methods. They align on conventions. And other developers violate those. Often times even conscientiously. How to handle those violations?
+
+Often times now is the time when the reviewer changes from consultant to finger-pointing judge (and hangman) who points at the wiki in which he documented the rule three years ago (without ever discussing it properly). Usually the tone gets rough. _Catch some breath, adjust your mindset..._.
```

Comment:
> Often times now is the time when the reviewer changes 

I'm not sure how you intended it, but here's an alternative ("often times" is incorrect) -

> Often when the reviewer changes ......... ; the tone gets rough.